python/cudf/cudf/core/udf/api.py

```diff
@@ -14,3 +14,11 @@ class Masked:
     def __init__(self, value, valid):
         self.value = value
         self.valid = valid
+
+
+def pack_return(masked_or_scalar):
+    # Blank function to give us something for the typing and
+    # lowering to grab onto. Just a dummy function for us to
```

Comment:
```suggestion
    # lowering to grab onto. Just a placeholder function for us to
```

https://developers.google.com/style/inclusive-documentation